pkgs/by-name/fr/freac/package.nix

```diff
@@ -3,37 +3,198 @@
   stdenv,
   fetchFromGitHub,
 
-  boca,
-  smooth,
+  boca, # Required for freac
+  smooth, # Required for boca
   systemd,
+
+  curl,
+  gnome-icon-theme,
+  openssl,
+
+  # Codecs
+  # exhale # Missing from nixpkgs as of 2025/07/21 ?
+  faac, # free = false
+  faad2,
+  fdk_aac,
+  ffmpeg,
+  flac,
+  lame,
+  libcdio,
+  libcdio-paranoia,
+  # libcdrip, # Missing from nixpkgs as of 2025/07/21 ?
+  libogg,
+  libopus,
+  libsamplerate,
+  libsndfile,
+  libvorbis,
+  # mac, # Same APE codec as monkeysAudio ?
+  monkeysAudio,
+  mp4v2,
+  mpg123,
+  # musepack, # Missing from nixpkgs as of 2025/07/21 ? or libmpcdec ?
```

Comment:
Might this be https://github.com/NixOS/nixpkgs/blob/a421ac6595024edcfbb1ef950a3712b89161c359/pkgs/by-name/li/libmpcdec/package.nix ?